env

```diff
@@ -0,0 +1 @@
+OPENAI_API_KEY=
```

Comment:
Would you mind adding a config/examples dir and populating with sample.env, including this var? I don't want anyone accidentally committing keys, and an examples dir can be easily referenced in our docs